fastplotlib/graphics/line_collection.py

```diff
@@ -30,6 +29,41 @@ def __init__(
             *args,
             **kwargs
     ):
+        """
+        Create a Line Collection
+
+        Parameters
+        ----------
+        data: list of array-like
+            List of line data to plot, if elements 2D must be of shape [n_points, 2], if elements
+            3D must be of shape [n_points, 3]
+
+        z_position: list of float or float, optional
+            z-axis position for each line graphic, single value will apply to
```

Comment:
be clearer, put each condition on a line and use `|` to denote the start of a line:

```rst
| if ``float``, single position will be used for all lines
| if ``list`` of ``float``, each value will apply to individual lines.
```